package.json

```diff
@@ -12,15 +12,19 @@
     "@testing-library/jest-dom": "^6.1.4",
     "@testing-library/react": "^14.0.0",
     "@testing-library/user-event": "^14.5.1",
+    "chart.js": "^4.4.0",
     "chokidar": "^3.5.3",
     "css-select": "^5.1.0",
+    "faker": "^6.6.6",
```

Comment:
just FYI I am getting a weird audit report:
![image](https://github.com/Expenses-tracker-app/frontend/assets/38724183/9845b535-4826-4a31-80b0-e598bd34d1cc)
it wants to install `faker@5.5.3` even though ˛faker@6.6.6.` is the latest 🧐

Reply:
I will delete it, it's old dependency